crates/cli/src/subcommands/call.rs

```diff
@@ -49,6 +49,14 @@ pub fn cli() -> clap::Command {
                 .action(ArgAction::SetTrue)
                 .help("If this flag is present, the call will be executed with no identity provided"),
         )
+        .arg(
+            Arg::new("subscribe")
+                .long("subscribe")
+                .long_flag_alias("follow")
+                .short('f')
```

Comment:
I think adding aliases to new commands is unnecessary and redundant, because we don't have a history to preserve.